web/files/files_test.go

```diff
@@ -533,6 +541,22 @@ func TestUploadWithMetadata(t *testing.T) {
 	assert.Equal(t, "2017-04-22T01:00:00-05:00", meta["datetime"])
 }
 
+func TestUploadWithSourceAccount(t *testing.T) {
+	buf := strings.NewReader("foo")
+	account := "0c5a0a1e-8eb1-11e9-93f3-934f3a2c181d"
+	identifier := "11f68e48"
+	req, err := http.NewRequest("POST", ts.URL+"/files/?Type=file&Name=with-sourceAccount&SourceAccount="+account+"&SourceAccountIdentifier="+identifier, buf)
+	req.Header.Add(echo.HeaderAuthorization, "Bearer "+token)
+	assert.NoError(t, err)
+	res, obj := doUploadOrMod(t, req, "text/plain", "rL0Y20zC+Fzt72VPzMSk2A==")
+	assert.Equal(t, 201, res.StatusCode)
+	data := obj["data"].(map[string]interface{})
+	attrs := data["attributes"].(map[string]interface{})
+	fcm := attrs["cozyMetadata"].(map[string]interface{})
+	assert.Equal(t, account, fcm["sourceAccount"])
+	assert.Equal(t, identifier, fcm["sourceAccountIdentifier"])
+}
+
```

Comment:
Is it possible to add a test when we create a file but the client already provided a few cozyMetaData ? 

Because we already provide a few cozyMetaData here (https://github.com/cozy/cozy-client/blob/046810a2143d5460d63b710ec543859b9d300056/packages/cozy-client/src/CozyClient.js#L304) and I don't know if we need to keep the createdAt attribute for a file for instance?

Reply:
The client cannot provide a few cozyMetadata when creating a file. There are only `sourceAccount` and `sourceAccountIdentifier` that can be given. The other fields are controlled by the stack.

For example, `cozyMetadata.createdAt` means the date/time of creation of the file on the server.

Reply:
OK thanks for the clarification !